### ISSUE TITLE
The way that payload-type:application/vnd.excalidraw+json serialization works in SVG files breaks their compatibility with screen readers

### ISSUE THREAD

**Issue by enzoferraribf** [excalidraw/excalidraw#7338]
Hi folks,

Screen readers are unable to bypass whatever is inside 

```
payload-type:application/vnd.excalidraw+json
``` 

when exporting to SVG. 

**That severily impacts accessibility for visual impaired people**. Screen readers will read it as a giant blob of text.

My tests were made in the Edge browser.

I'm willing to work on this issue if maintainers find it interesting.

Thanks.

**Comment by dwelle** [comment]
Sorry, missed this issue. 

> Valid would be to put such payload in a comment, or use the `<metadata/>` tag.

I'm not sure why we didn't. PR welcome!
